packages/frontend/src/utils/usePageTitle.tsx

```diff
@@ -9,11 +10,30 @@ interface PageTitleOptions {
   getFilterLabel?: (key: string, values: (string | number)[]) => string | undefined;
 }
 
+const translateInboxViewType = (baseTitle: string, t: TFunction): string => {
```

Comment:
We already have an Enum defined against the viewTypes called `InboxViewType`. I suggest using this instead of string and change type of `baseTitle` to `InboxViewType`...

Instead of 
```
    const translatedBaseTitle = translateInboxViewType(baseTitle, t);
```

you now can do this:

```
    const translatedBaseTitle =  t(`sidebar.sent.${baseTitle}`)
```